src/components/Card/Card.scss

```
@@ -0,0 +1,74 @@
+.card {
+  max-width: 306px;
+  height: 502px;
+  
+  * + * {
+    padding-left: 47px;
+    padding-right: 47px;
+  }
+
+  & h1 {
+    color: white;
+    font-size: 32px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  & p {
+    padding-top:15px;
+    font-size: 18px;    
+    line-height: 1.3em;
+  }
+
+  & button {

```

same here: https://github.com/MarcelloAragoni/mentor/pull/2/files#r844145043

```suggestion
  button {

```